tests/renderer/task-runner-spec.tsx

```diff
@@ -0,0 +1,189 @@
+import {
+  BisectRequest,
+  ElectronReleaseChannel,
+  RunResult,
+  RunnableVersion,
+  TestRequest,
+  VersionSource,
+  VersionState,
+} from '../../src/interfaces';
+import { IpcEvents } from '../../src/ipc-events';
+
+import { App } from '../../src/renderer/app';
+import { Runner } from '../../src/renderer/runner';
+import { AppState } from '../../src/renderer/state';
+import { TaskRunner } from '../../src/renderer/task-runner';
+import { IpcRendererManager } from '../../src/renderer/ipc';
+
+describe('Task Runner component', () => {
+  let app: Partial<App>;
+  let appState: Partial<AppState>;
+  let runner: Partial<Runner>;
+  let ipc: IpcRendererManager;
+  // let instance: TaskRunner;
```

Comment:
Is this comment intentional here?

Reply:
No. Removed in f58d1467da956ac11c1da63555b6bc9faa8f55e2